tests/test_stability_funs.py

```diff
@@ -429,11 +429,7 @@ def test_ballooning_geometry(tmpdir_factory):
             "g^rz",
         ]
 
-        grid = Grid.create_meshgrid(
-            [rho, alpha, zeta],
-            coordinates="raz",
-            period=(np.inf, 2 * np.pi, np.inf),
```

Comment:
why get rid of `period` on all of these calls?

Reply:
The mapping from flux coords to field line coordinates is secular. If you enforce periodicity the mapping is in general wrong. See #1180. For this application, I remember I showed that it doesn't matter before we merged the original pull request for ballooning stability. Still I don't like it, because its basically a foot-gun waiting to happen and I have yet to see it converge any faster in the root finding.

Reply:
Removing is fine with me but this is all 1D root finding and not 2D interpolation in alpha, zeta so it seems our results won't change. In practice, we always keep alpha in [0, pi]